tests/test_docker_compose.py

```diff
@@ -23,6 +25,18 @@ def test_can_pull_images_before_spawning_service_via_compose():
         assert port == "4444"
 
 
+def test_can_build_images_before_spawning_service_via_compose():
+    with patch.object(DockerCompose, DockerCompose._call_command.__name__) as call_mock:
```

Comment:
No need to use `DockerCompose._call_command.__name__`, just use `"_call_command"`

Reply:
There is good reason to use `DockerCompose._call_command.__name__` vs. `"_call_command"` and that, when using a typical IDE to refactor the method name, most IDEs will update the test automatically using the former code, and cause the test to break using the suggestion.  

But I don't feel that strongly about it so I'll change it now.